code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -726,25 +726,26 @@
 // called from mob/living/carbon/human/Crossed()
 // when mulebot is in the same loc
 /mob/living/simple_animal/bot/mulebot/proc/RunOver(mob/living/carbon/human/H)
-	log_combat(src, H, "run over", null, "(DAMTYPE: [uppertext(BRUTE)])")
-	H.visible_message("<span class='danger'>[src] drives over [H]!</span>", \
-					"<span class='userdanger'>[src] drives over you!</span>")
-	playsound(src, 'sound/effects/splat.ogg', 50, TRUE)
-
-	var/damage = rand(5,15)
-	H.apply_damage(2*damage, BRUTE, BODY_ZONE_HEAD, run_armor_check(BODY_ZONE_HEAD, MELEE))
-	H.apply_damage(2*damage, BRUTE, BODY_ZONE_CHEST, run_armor_check(BODY_ZONE_CHEST, MELEE))
-	H.apply_damage(0.5*damage, BRUTE, BODY_ZONE_L_LEG, run_armor_check(BODY_ZONE_L_LEG, MELEE))
-	H.apply_damage(0.5*damage, BRUTE, BODY_ZONE_R_LEG, run_armor_check(BODY_ZONE_R_LEG, MELEE))
-	H.apply_damage(0.5*damage, BRUTE, BODY_ZONE_L_ARM, run_armor_check(BODY_ZONE_L_ARM, MELEE))
-	H.apply_damage(0.5*damage, BRUTE, BODY_ZONE_R_ARM, run_armor_check(BODY_ZONE_R_ARM, MELEE))
-
-	var/turf/T = get_turf(src)
-	T.add_mob_blood(H)
-
-	var/list/blood_dna = H.get_blood_dna_list()
-	add_blood_DNA(blood_dna)
-	bloodiness += 4
+	if(wires.is_cut(WIRE_AVOIDANCE))
```

Comment:
Early return to avoid horizontal drift, please.

Reply:
Sorry, I'm still learning the basics of coding byond. Could you explain what you mean please?

Reply:
this code...
```dm
if(thing)
	code()
	code()
	if(other_thing)
		code()
```
...becomes this with early returns:
```dm
if(!thing)
	return
code()
code()
if(!other_thing)
	return
code()
```
Makes sense?

Reply:
Ah, thanks. I _think_ I fixed it now?